x-pack/plugins/ml/public/maps/anomaly_source.tsx

```diff
@@ -198,12 +204,30 @@ export class AnomalySource implements IVectorSource {
   }
 
   async getImmutableProperties(): Promise<ImmutableSourceProperty[]> {
+    let explorerLink: string | undefined;
+
+    try {
+      explorerLink = await AnomalySource.mlLocator?.getUrl({
+        page: ML_PAGES.ANOMALY_EXPLORER,
+        pageState: {
+          jobIds: [this._descriptor.jobId],
+          timeRange: {
+            ...(AnomalySource.timefilter?.getTime() || {}),
```

Comment:
Looks like the time range is set when the edit layer panel is opened. Is it possible to update the range if the user changes the time picker while the flyout is open?